_posts/2019-05-02-largest-area-between-points.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Largest Area Between Points (Advent of Code 2018)
+category: "AlgoSIG 3"
+link: https://adventofcode.com/2018/day/6
+author:
+tags:
+  - unsolved
+---
+
+## Description
+
+Using only the Manhattan distance, determine the area around each coordinate by counting the number of integer X,Y locations that are closest to that coordinate (and aren't tied in distance to any other coordinate). Your goal is to find the size of the largest area that isn't infinite.
+
+You'll need to get your own input for this challenge from [this Advent of Code problem]({{ post.link }}).
```

Comment:
Same here - {{ page.link }}

Reply:
post.link did work, but page.link is more consistent